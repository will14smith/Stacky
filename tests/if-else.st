main () -> () { 1 false { 2 + } { 1 + } if-else print }