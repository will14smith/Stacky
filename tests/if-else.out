=== Output ===
2

=== Exit ===
0
